com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/DefaultPOSerializer.java

```diff
@@ -173,20 +193,46 @@ public PO fromJson(JsonObject json, PO po) {
 				field = json.get(columnName);
 			if (field == null)
 				continue;
-			if (!column.isUpdateable())
-				continue;
-			if (column.isSecure() || column.isEncrypted())
-				continue;
+			boolean errorOnNonUpdatable = MSysConfig.getBooleanValue("REST_ERROR_ON_NON_UPDATABLE_COLUMN", true);
+			boolean allowUpdateSecure = MSysConfig.getBooleanValue("REST_ALLOW_UPDATE_SECURE_COLUMN", true);
+			if (!column.isUpdateable()) {
+				if (errorOnNonUpdatable)
```

Comment:
Hi, this if statement with the same logic and a slightly different message is written many times in this commit. I think it would be better to create a method for that

Reply:
ready, refactored and tested